mentors/python/e.json

```
@@ -1 +1,8 @@
-[]
+[{
```

```suggestion
[
  {
```
Small formatting changes.

---

Thanks for the help :)